### PR TITLE
Added my yaml.

### PR DESCRIPTION
**Pull request by cognettings**
Just adding my yaml here.

### CODE REVIEW

File: scripts/people/cognettings.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Comments are a thing?
+- blog: http://cognettings.wordpress.com
+- feed: http://cognettings.wordpress.com/feed
```
Comment (decause) [inline scripts/people/cognettings.yaml]: There seems to be an extra dash here. Remove it, and I'll merge.

**Comment by cognettings** [comment]
Okay, it should be fixed now.

**Comment by cognettings** [comment]
I forgot to push changes to GitHub before. They should be there now.

**Comment by cognettings** [comment]
This now also contains my litreview1 link.

**Comment by cognettings** [comment]
Added quiz1

**Comment by decause** [comment]
:wrench: Fixed!

**Pull request merged**